cluster_assign_roles.rb

```diff
@@ -420,6 +421,26 @@ def ssh(host:, username:, password:, command:, streaming: false)
     end
   end
 
+  def stop_all_clients
+    target_nodes = parse_cluster_txt.reject do |entry|
+      entry[:runlist].include?('SKIP')
+    end
+
+    target_nodes.each do |node|
+      result = ssh(host: node[:ip_address],
+                   username: 'ubuntu',
+                   password: cobbler_root_password,
+                   command: 'killall -9 chef-client',
```

Comment:
Probably needs to have `sudo` at the beginning. Might not hurt to also verify no `chef-client` processes are lingering after this, even though a `kill -9` should get the job done.